lib/rihanna/job.ex

```diff
@@ -179,6 +179,28 @@ defmodule Rihanna.Job do
     end
   end
 
+  @doc false
+  def delete(pg \\ Rihanna.Job.Postgrex, job_id) do
+    result =
+      Postgrex.query!(
+        pg,
+        """
+          DELETE FROM "#{table()}"
+          WHERE
+            id = $1
+        """,
+        [job_id]
+      )
+
+    case result.num_rows do
+      0 ->
+        {:error, :job_not_found}
+
+      1 ->
+        {:ok, :deleted}
```

Comment:
It might be nice to return the deleted job. You can do this using `RETURNING`

Reply:
Something like?
```suggestion
        {:ok, id}
```
or
```suggestion
        {:ok, job}
```

Reply:
Nice, I was able to implement